spyrograph/_trochoid.py

```diff
@@ -68,12 +68,10 @@ def __init__(
         self.thetas = self._validate_theta(thetas, theta_start, theta_stop, theta_step)
         self.origin = origin
    
-        while deci != False:
-            if self.R <= 0 or self.r <= 0 or self.d <= 0:
-                raise ValueEror("Negative and/or zero input parameters were passed. Please only pass positive values")
-                deci = True
-            else:
-                deci = False
+        if self.R <= 0 or self.r <= 0 or self.d <= 0:
+            raise ValueEror("Negative and/or zero input parameters were passed. Please only pass positive values")
+        else:
+            Pass
```

Comment:
Okay one last change!!! 

All we need is the `if` statement - this `else` statement doesn't need to be here